opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -11,20 +11,64 @@ use std::{cell::RefCell, str, time::SystemTime};
 
 thread_local! { static EBW: RefCell<EventBuilder> = RefCell::new(EventBuilder::new());}
 
-/// UserEventsExporter is a log exporter that exports logs in EventHeader format to user_events tracepoint.
-pub(crate) struct UserEventsExporter {
-    provider: Mutex<Provider>,
-    name: String,
-    event_sets: Vec<Arc<EventSet>>,
-    cloud_role: Option<String>,
-    cloud_role_instance: Option<String>,
+/// Options for configuring the User Events Exporter.
+#[derive(Debug)]
+pub struct ExportOptions<'a> {
```

Comment:
I'm okay with having lifetime in the struct. If we still want to simplify, we could do something like this:

```rust
pub struct ExportOptions<T: AsRef<Str>> {
    provider_name: T
}